e2e_demo/scala/dl_dependencies.sh

```diff
@@ -17,6 +17,11 @@ if [ ! -f ${SPARK3_DETAILS}.tgz ]; then
   wget https://archive.apache.org/dist/spark/spark-3.3.1/${SPARK3_DETAILS}.tgz &
 fi
 wait
+
+
+########################################################################
+# Setting variables
+########################################################################
```

Comment:
This is actually just extracting the artifacts (if needed) not setting variables.

Reply:
Brainfart. Too much copy-pasta